### PR TITLE
add models for bartender

### PR DESCRIPTION
**Pull request by joshuahefner**
see azurestandard/api-spec#167
and azurestandard/beehive#2321

### CODE REVIEW

File: azure-providers.js [hunk 1]
```diff
@@ -305,6 +312,17 @@ var azureProvidersModule = angular
...
                 }
+                if (model === 'bartender-print-configuration') {
+                    actions.testPrint = {
+                        method: 'POST',
+                        url: url + '/' + model + '/:' + identifier + '/actions/test-print',
+                        withCredentials: true,
+                        headers: _headers,
+                        transformRequest: function(data, headersGetter) {
+                          return null;
+                        },
```
Comment (wking) [inline azure-providers.js]: What's the `transformRequest` bit for?

Comment (MercenaryJosh) [inline azure-providers.js]: The api expects there to not be a body on the post request to `/actions/test-print` and will return an error is there is one. Angular expects that all POST requests will have a body, the transformRequest method here strips out the body before the request is sent.

**Comment by wking** [comment]
On Thu, Jan 26, 2017 at 09:33:47AM -0800, David McAtee wrote:
> @wking can you confirm that this is ready to be merged?

I just pushed some minor fixups with 851b726 → 85c5831.  That catches
this up with the currently-deployed API, but we'll need some work on
testPrint to adjust to the in-flight azurestandard/api-spec#190.  I
don't mind if that lands in this PR or via a new one.

**Comment by igregson** [comment]
@wking and @davidmcatee-azure 

Can go ahead and merge this? This is blocking [internal-website #178](https://github.com/azurestandard/internal-website/pull/178).

**Comment by davidmcatee-azure** [comment]
We might as well update this to support azurestandard/api-spec#190. We need to get rid of the `transformRequest` bit [mentioned above](https://github.com/azurestandard/azure-angular-providers/pull/63#discussion_r92729452). I'll try to get to it after putting the kids to bed.

**Pull request merged**